webpack.config.js

```diff
@@ -138,5 +140,6 @@ module.exports = {
   plugins,
   resolve: {
     alias: { path: false },
+    modules: ['src', 'node_modules', 'node_modules/govuk_frontend_toolkit/images'],
```

Comment:
cunning fix to correct css-loader upgrade problems not being able to find toolkit files